proxmox-via-prometheus.json

```diff
@@ -592,11 +603,48 @@
             ]
           }
         },
-        "overrides": []
+        "overrides": [
+          {
+            "matcher": {
+              "id": "byRegexp",
+              "options": "/.*pvenode1/"
+            },
+            "properties": [
+              {
+                "id": "displayName",
+                "value": "pnenode1"
+              }
+            ]
+          },
+          {
+            "matcher": {
+              "id": "byRegexp",
+              "options": "/.*pvenode2/"
+            },
+            "properties": [
+              {
+                "id": "displayName",
+                "value": "pvenode2"
+              }
+            ]
+          },
+          {
+            "matcher": {
+              "id": "byRegexp",
+              "options": "/.*pvenode3/"
+            },
+            "properties": [
+              {
+                "id": "displayName",
+                "value": "pvenode3"
+              }
+            ]
+          }
+        ]
```

Comment:
It looks to me that the overrides you added will only be working for a cluster of three nodes named pvenode1-3. You may want to remove those from the pull request, since they are not universal to every proxmox cluster.